src/main/java/com/minecolonies/coremod/entity/ai/citizen/herders/AbstractEntityAIHerder.java

```diff
@@ -502,19 +499,5 @@ private void butcherAnimal(final EntityAnimal animal)
      *
      * @return the {@link Item} needed for breeding the animal.
      */
-    public Item getBreedingItem()
-    {
-        if (getAnimalClass().equals(EntityChicken.class))
-        {
-            return Items.WHEAT_SEEDS;
-        }
-        if (getAnimalClass().equals(EntityPig.class))
-        {
-            return Items.CARROT;
-        }
-        else
-        {
-            return Items.WHEAT;
-        }
-    }
+    public abstract Item getBreedingItem();
```

Comment:
ItemStack not Item!. This gets complicated when we later expose this in our API.

Reply:
done